WIP/calculator.rb

```diff
@@ -0,0 +1,95 @@
+class Calculate
+
+  def initialize(**args)
+    @accumulate = args[:accumulate] || args[:accumulate] == nil
```

Comment:
Were we going to come up with a better name for this variable that is more descriptive of what it does? or are we happy with this name? 

Let's also add a comment in the code to say what the default value here will be if the user does not pass in any value.

Reply:
Sorry couldn't find a better name.